kubernetes/cray-ims/templates/cray-ims-rbac.yaml

```diff
@@ -72,6 +72,7 @@ metadata:
   name: ims-service-job-mount
   namespace: {{ .Values.ims_config.cray_ims_job_namespace }}
 ---
```

Comment:
This ServiceAccount is also not needed when the following ClustRoleBinding isn't used.